llmfoundry/command_utils/eval.py

```diff
@@ -345,7 +328,6 @@ def evaluate(cfg: DictConfig) -> tuple[list[Trainer], pd.DataFrame]:
              device_eval_batch_size=eval_config.device_eval_batch_size,
              eval_gauntlet_config=eval_gauntlet_config,
              eval_loader_config=eval_loader_config,
-             fsdp_config=fsdp_config,
```

Comment:
shouldn't we be passing in `parallelism_config` isntead?

Reply:
ah yeah, looks like that wasn't added before, but should have been?

Reply:
oops yes, added. good catch!